KarhooUISDK/Workers/TripInfoUtility.swift

```diff
@@ -56,17 +56,15 @@ public struct TripInfoUtility {
                                                  .driverEnRoute: UITexts.GenericTripStatus.enRoute,
                                                  .arrived: UITexts.GenericTripStatus.arrived,
                                                  .passengerOnBoard: UITexts.GenericTripStatus.passengerOnBoard,
-                                                 .completed: UITexts.GenericTripStatus.completed,
-                                                 .unknown: UITexts.GenericTripStatus.unkown,
-                                                 .incomplete: UITexts.GenericTripStatus.incomplete]
+                                                 .completed: UITexts.GenericTripStatus.completed]
         return tripStatuses[tripState] ?? ""
     }
 
     public static func longDescription(trip: TripInfo) -> String {
         let requestedState = String(format: UITexts.Trip.tripStatusRequested,
                 trip.fleetInfo.name)
 
-        let tripStatuses: [TripState: String] = [.unknown: UITexts.Trip.tripStatusUnkown,
```

Comment:
Why was this removed?

Reply:
UITexts.Trip.tripStatusUnkown didn't exist so the unknown status was never showing anything and it is not existing on android